assets/js/analysis/show.controller.js

```diff
@@ -519,7 +519,26 @@
 					 {
 							 wordspace.style.fontWeight = 'bold';
 							 wordspace.setAttribute("class", word.ner.toLowerCase());
-					 }
+           }
+
+           if(type == 'relation') {
+             var relationTitle = '';
+
+             $scope.analysis.result.sentences[sk].relations.forEach(function(relation) {
+               var startInd = relation.subject.start;
+               var endInd = relation.object.end - 1;
+               
+               if(wk >= startInd && wk <= endInd) {
+                 wordspace.style.fontWeight = 'bold';
+                 wordspace.style.textDecoration = 'underline';
+
+                 var relationTriple = relation.subject.lemma + ' (' + relation.relation.lemma + ') ' + relation.object.lemma;
+                 relationTitle += relationTriple + '\n'; 
+               }
+             });
+
+             wordspace.setAttribute('title', relationTitle);
```

Comment:
My worry here is like, what if there's another relationship triple in the middle of a long relationship triple?  The code here would overwrite the old tooltip with the new relationship triple.  I'd say we could either only highlight the words in the triple or try to add to the tooltip rather than replace it here.